rcppsmc.Rproj

```diff
@@ -14,3 +14,4 @@ LaTeX: pdfLaTeX
 
 BuildType: Package
 PackageInstallArgs: --no-multiarch --with-keep.source
+PackageRoxygenize: rd,collate,namespace,vignette
```

Comment:
I generally do not let my packages do anything but the rd part.

Please refrain from build changes without first establishing agreement.